react-validated-input/my-app/src/PassInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { FaCheck } from 'react-icons/fa';
+import { FaXmark } from 'react-icons/fa6';
+
+export function PassInput() {
+  const [passWord, setPassWord] = useState('');
+
+  function changePasswordInput(val: string) {
+    setPassWord(val);
+  }
+  if (passWord.length === 0) {
```

Comment:
The if/else if/else works, but a better solution uses conditional rendering, as we did in the Q&A. That way you don't have so much duplicate code.